example/basic/example.dart

```diff
@@ -21,10 +21,13 @@ main() {
         name: 'one',
         defaultRoute: true,
         path: '/one',
-        pageTitle: 'Route One',
+        pageTitle: (_) => 'Route One',
```

Comment:
Would you be open to making it dynamic (wish Dart had union types) so that it could be a String or a `PageTitleHandler` function? Just like the `ref` prop on UI components.